modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/DatePicker/DatePicker.es.js

```diff
@@ -235,6 +235,21 @@ export default function DatePicker({
 		}
 	};
 
+	const [expanded, setExpanded] = useState(false);
+
+	const handleExpandedChange = (value) => {
+		if (value !== expanded) {
+			setExpanded(value);
```

Comment:
@marcosapmf in this case, expanded should always be updated regardless of the business rule, so you can call `setExpanded(value);` out of condition.

Reply:
On second thought, this condition is not necessary, `onExpandedChange` works like a toggle, it will be called to invert the state. Did you find any special use cases?